Truncate error messages in passportAuth redirectUri (#3596)
blitz-js/blitz: .changeset/breezy-moose-behave.md

---
"blitz": minor
"@blitzjs/auth": patch
---

> Error messages representing errors that are thrown while handling the `api/auth/<strategy>/callback` request, will be truncated to a maximum of 100 characters before being passed as to the `?authError=` query parameter of the redirectUrl.

Reviewer: ```suggestion
Truncate errors from `api/auth/<strategy>/callback` request to 100 characters before passing them to the `?authError=` query parameter
```
